### PR TITLE
SOL-1926: Org filter on courses in otto.

### PR DESCRIPTION
**Pull request by malikshahzad228**
Add a description of your changes with links to any relevant material.
### Reviewers

Possible roles follow. The PR submitter checks the boxes after each reviewer finishes and gives :+1:. 
- [ ] Subject matter expert: @mattdrayer @saleem-latif 
- [ ] Doc team review (sanity check/copy edit/dev edit): @catong @lamagnifica @pdesjardins @srpearce

FYI: Tag anyone else who might be interested in this PR here.
### Testing
- [x] Ran ./run_tests.sh without warnings or errors
### Post-review
- [x] Squash commits

### CODE REVIEW

**Comment by lamagnifica** [comment]
https://openedx.atlassian.net/browse/SOL-1926

File: en_us/install_operations/source/ecommerce/install_ecommerce.rst [hunk 1]
```diff
@@ -204,6 +204,10 @@ this command.
      - No
      - Name of the partner.
      - ``--partner-name='Open edX'``
+   * - ``--partner-org``
+     - No
+     - Organizations of the partner.
+     - ``--partner-name=foo,bar``
```
Comment (mattdrayer) [inline en_us/install_operations/source/ecommerce/install_ecommerce.rst]: I think should this be `--partner-org=foo,bar`

File: en_us/install_operations/source/ecommerce/create_products/create_course_seats.rst [hunk 2]
```diff
@@ -22,6 +22,11 @@ To create a course seat, follow these steps.
    * **Course ID**
    * **Course Name**
 
+   .. note::
+
+     Course must be from one of the organization which are mention 
+     in :ref:`Configure a Site Partner and Site Configuration` section.
```
Comment (pdesjardins) [inline en_us/install_operations/source/ecommerce/create_products/create_course_seats.rst:28]: The course must be from one of the partner organizations that you have configured for your E-Commerce service. For more information about configuring partner organizations, see :ref:`Configure a Site Partner and Site Configuration`.

**Comment by pdesjardins** [comment]
The software changes that this doc update are for has been reverted. We don't need to make this update. In case organization is reintroduced, here's a wiki page that describes what they are.

https://openedx.atlassian.net/wiki/display/AC/Open+edX+Partners%2C+Sites%2C+and+Organizations

@mattdrayer, thanks for clarifying!

**Pull request closed without merging**